package.json

```diff
@@ -14,7 +14,7 @@
     "lint:fix": "yarn lint --fix",
     "create-modular-react-app": "ts-node packages/create-modular-react-app/src/cli.ts",
     "modular": "ts-node packages/modular-scripts/src/cli.ts",
-    "test": "yarn modular test --watchAll false --runInBand",
+    "test": "yarn modular test --watchAll false --runInBand --env node",
```

Comment:
Our tests are mostly back end tests that don't require the JSDOM env that we set as default for Modular (Modular  users will continue to use JSDOM as default) 
Modular tests that build/run front end components have been changed to specify JSDOM as the env (ignoring the flag passed here)